### PR TITLE
[BAAI]add 5 stdcase of base: FP64-benchmark, FP64-toolkit, BF16-benchmark, BF16-toolkit and INT8-toolkit

### PR DESCRIPTION
**Pull request by ljy-2000**

### CODE REVIEW

File: base/toolkits/computation-INT8/nvidia/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## 其他重要监控结果
+
+| 监控项  | 系统平均CPU占用 | 系统平均内存占用 | 单卡平均温度  | 单卡平均显存占用 |
+| ---- | --------- | -------- | ------- | -------- |
+| 监控结果 | 0.643%    | 1.085%   | 49.23°C | 1.876%   |
+
+# 厂商测试工具原理说明
+
+使用GEMM算子进行computation-bound的计算任务，从而测得实际INT8算力
```
Comment (shh2000) [inline base/toolkits/computation-INT8/nvidia/README.md:42]: 这里要补充一下INT8的作用，以及为什么BLAS只有8*8=32的接口
随后从8*8=32推导出INT8 GEMM不只使用了INT8 TC，因此虽标定624，但INT8 GEMM的theoretical peak不是624。此case评测结果仅供参考等等

Comment (ljy-2000) [reply base/toolkits/computation-INT8/nvidia/README.md:42]: 好的，我添加一下

**Comment by shh2000** [comment]
please add FP64, BF16, and INT8 to FlagPerf/README.md, you can refer to FP32 and TF32

**Comment by ljy-2000** [comment]
> please add FP64, BF16, and INT8 to FlagPerf/README.md, you can refer to FP32 and TF32
> 
> the order can be FP64, FP32, TF32, Fp16, BF16, INT8. from more bits to less bits, from IEEE 754 standard to AI domain



> please add FP64, BF16, and INT8 to FlagPerf/README.md, you can refer to FP32 and TF32
> 
> the order can be FP64, FP32, TF32, Fp16, BF16, INT8. from more bits to less bits, from IEEE 754 standard to AI domain

OK, shall I open a new pull request or modify FLAGPERF/README.md together in #514?

**Review by shh2000 — APPROVED** [review]

**Review by upvenly — APPROVED** [review]

**Pull request merged**